CommandLine.c

```diff
@@ -231,6 +235,9 @@ static CommandLineSettings parseArguments(const char* program, int argc, char**
             flags.highlightChanges = true;
             break;
          }
+         case 128:
+            flags.readonly = true;
+            break;
```

Comment:
Braces are needed if a variable is declared inside the top level of the case block.
Since this is not the case for this block I'd like to not enclose it.

Reply:
There are other blocks in this case statement that have them despite not needing them. That's why I'm asking.